src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -453,8 +455,7 @@ public void ProcessStart_UseShellExecuteTrue_OpenUrl_SuccessfullyReadsArgumentAr
             }
         }
 
-        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
-        [Trait(XunitConstants.Category, XunitConstants.RequiresElevation)]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsUnixAndSuperUser))]
```

Comment:
It no longer checks `RemoteExecutor.IsSupported`, is that ok?
Also, you already know is Unix because `ProcessTests.Unix.cs` only compiles for it.